### PR TITLE
msc-generator 7.3

### PR DESCRIPTION
**Pull request by milyenpabo**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by milyenpabo** [comment]
Hi maintainers, any ideas on what went wrong? I don't see any specific error under the failed job.

**Comment by SMillerDev** [comment]
```
==> ./configure --prefix=/opt/homebrew/Cellar/msc-generator/7.3 --disable-font-checks
Failed to execute: ./configure
```

**Comment by milyenpabo** [comment]
Ahh, thanks, sorry, it turned out my browser firewall did not let the detailed logs to be viewed...

**Comment by milyenpabo** [comment]
Yes, we are working on this. Locally I could reproduce a configure failure ("no supported zip library found").

By the way, is there a method to access more detailed logs/error messages from the CI build? The message "Failed to execute: ./configure" is not utterly informative.

**Comment by cho-m** [comment]
> By the way, is there a method to access more detailed logs/error messages from the CI build? The message "Failed to execute: ./configure" is not utterly informative.

The failure is just due to missing `configure` in tarball. Since it isn't packaged with GitLab tarball, it needs to be generated first.

Once you add a step to do this in formula (e.g. running their helper script `./autogen` or directly running `autoreconf ...`) then stdout/stderr from `configure` should be output to CI logs and some additional logs will be included in uploaded GitHub artifacts.

**Comment by milyenpabo** [comment]
Makes sense, thanks @cho-m.

**Comment by milyenpabo** [comment]
@cho-m @SMillerDev Do you need anything else from my side to complete this PR?

**Comment by SMillerDev** [comment]
If autogen isn't needed it can be removed.

**Comment by milyenpabo** [comment]
@SMillerDev I removed the unused commands. The 11-arn64 CI job failed, but I don't see a reason. Maybe a timeout happend?

Is there a way to retrigger CI?

**Comment by milyenpabo** [comment]
@SMillerDev the checks are green now (I'm not sure what happened).

Please let me know if there's anything else you need for this PR.

File: Formula/msc-generator.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class MscGenerator < Formula
   desc "Draws signalling charts from textual description"
   homepage "https://sourceforge.net/p/msc-generator"
-  url "https://downloads.sourceforge.net/project/msc-generator/msc-generator/v7.x/msc-generator-7.2.tar.gz"
-  sha256 "40c7a45e1bce96bc93440b604eb0fd2b7894985fa0a723bf6483565bef026a2e"
+  url "https://gitlab.com/msc-generator/msc-generator/-/package_files/42434730/download"
```
Comment (SMillerDev) [inline Formula/msc-generator.rb:4]: Why did the source change? And if it needs to change, why not to https://gitlab.com/msc-generator/msc-generator/-/archive/v7.3.1/msc-generator-v7.3.1.tar.gz ?

File: Formula/msc-generator.rb [hunk 2]
```diff
@@ -34,8 +34,6 @@ class MscGenerator < Formula
 
   def install
     system "./configure", "--prefix=#{prefix}", "--disable-font-checks"
-    # Dance around upstream trying to build everything in doc/ which we don't do for now
-    # system "make", "install"
     system "make", "-C", "src", "install"
     system "make", "-C", "doc", "msc-gen.1"
```
Comment (SMillerDev) [inline Formula/msc-generator.rb:38]: can we do this here then?
```suggestion
    system "make"
    system "make", "install"
```

**Comment by milyenpabo** [comment]
The project repo has been migrated from Sourceforge to GitLab. The Sourceforge project will be abandoned soon. I updated the "homepage" field now to reflect the change.

I changed the URL to another one which contains the filename and version number. I removed the explicit version number from the formula.

File: Formula/msc-generator.rb (hunk 2 above)
Comment (milyenpabo) [reply Formula/msc-generator.rb:38]: Can we use simply `make install`? (And remove `make` from your suggestion?)

File: Formula/msc-generator.rb (hunk 1 above)
Comment (milyenpabo) [reply Formula/msc-generator.rb]: The project repo has been migrated from Sourceforge to GitLab. The Sourceforge project will be abandoned soon. I updated the "homepage" field now to reflect the change.

I changed the URL to another one which contains the filename and version number. I removed the explicit version number from the formula.

File: Formula/msc-generator.rb (hunk 2 above)
Comment (SMillerDev) [reply Formula/msc-generator.rb:38]: If that works, sure

Comment (milyenpabo) [reply Formula/msc-generator.rb:38]: We guessed it would work, but it didn't. I reverted this change.

**Comment by milyenpabo** [comment]
@SMillerDev I guess CI failed again with no apparent reason. Maybe a timeout?

**Comment by milyenpabo** [comment]
@SMillerDev can you retrigger CI? Or should I do that? If so, please let me know how to do that.

**Comment by milyenpabo** [comment]
@SMillerDev thanks, it passed now. Please let me know if the PR is ready now.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**